src/settings/global_user.rs

```diff
@@ -42,3 +45,15 @@ fn get_global_config() -> Result<GlobalUser, failure::Error> {
         }
     }
 }
+
+pub fn get_global_config_dir() -> Result<PathBuf, failure::Error> {
+    let home_dir = if let Ok(value) = env::var("WRANGLER_HOME") {
```

Comment:
do we (want to) indicate use of this env var in any of our documentation?

Reply:
Currently it's undocumented, once this lands we can create an issue. Using XDG standarts (#226) might changes how it works.